tests/scripts/basic-build-test.sh

```diff
@@ -25,13 +25,41 @@
 #
 # This script has been written to be generic and should work on any shell.
 #
-# Usage: basic-build-tests.sh
+# Usage: basic-build-tests.sh               to run all tests
+#        basic-build-tests.sh unit_test     to run only unit tests
 #
 
 # Abort on errors (and uninitiliased variables)
 set -eu
 
-if [ -d library -a -d include -a -d tests ]; then :; else
+CONFIG_H='include/mbedtls/config.h'
+CONFIG_BAK="$CONFIG_H.bak"
+
+IS_UNIT_TEST=0
```

Comment:
I'm not sure I like the variable name here, I don't see what the "is" refers to. How about `UNIT_TESTS_ONLY` instead?

Reply:
Changed as suggested